tests/test_commands.py

```diff
@@ -79,6 +80,15 @@ def kill_proc():
 
         return p, to_unicode(stdout), to_unicode(stderr)
 
+    def find_in_file(self, filename, regex):
+        '''Find first pattern occurrence in file'''
```

Comment:
```suggestion
        """Find first pattern occurrence in file"""
```

:lipstick: I think this is the only area were PEP8 actually makes a choice between `'` and `"` :slightly_smiling_face: